### PR TITLE
Rename to cryostat-core

### PR DESCRIPTION
**Pull request by andrewazores**

### CODE REVIEW

File: src/main/java/io/cryostat/core/net/Credentials.java [hunk 1]
```diff
@@ -39,26 +39,26 @@
...
 
-    String getUsername() {
+    public String getUsername() {
         return username;
     }
 
-    String getPassword() {
+    public String getPassword() {
         return password;
     }
```
Comment (ebaron) [inline src/main/java/io/cryostat/core/net/Credentials.java:63]: Is this visibility change intentional?

**Review by ebaron — COMMENTED** [review]
Looks good, just one question about a non-renaming change.

File: src/main/java/io/cryostat/core/net/Credentials.java (hunk 1 above)
Comment (andrewazores) [reply src/main/java/io/cryostat/core/net/Credentials.java:63]: No, that looks like a mistake - not sure how that snuck in. Thanks for catching it, very sharp eye!

Comment (andrewazores) [reply src/main/java/io/cryostat/core/net/Credentials.java:63]: (that change is used for something else in a testing branch related to autorules, but I don't know how it ended up in this patch)

**Review by ebaron — APPROVED** [review]

**Pull request merged**